src/styles/blocks/grid.scss

```diff
@@ -0,0 +1,57 @@
+.grid {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  justify-items: center;
+  column-gap: 20px;
+
+  @include onTablet {
+    grid-template-columns: repeat(6, 1fr);
+    column-gap: 24px;
+  }
+
+  @include onDesktop {
+    grid-template-columns: repeat(12, 1fr);
+  }
+
+  &--desktop {
+    display: block;
+
+
+    @include onTablet {
+      display: grid;
+    }
+  }
```

Comment:
Weird. The grid that is not a grid))
If you want the grid to occupy full-width better use `grid-template-columns: 1fr;`.